DL_AdditionalMessage.cpp

```diff
@@ -0,0 +1,143 @@
+#include <iostream>
+#include <string>
+
+
+enum class MessageIntentUse {
+     DueToWeather,
+     DueToAircraft
+    
+};
+
+enum class MessageElement {
+    URG,
+    ALRT,
+    RESP
+};
+
+
+class Message {
```

Comment:
here u have to use the correct name like DL_AdditionalMessage
`class DL_AdditionalMessage: public DataLinkMessage`

Reply:
I rename the class Name